specification/containerservice/Fleet.Management/tspconfig.yaml

```diff
@@ -4,6 +4,7 @@ options:
   '@azure-tools/typespec-autorest':
     azure-resource-provider-folder: "resource-manager"
     emitter-output-dir: "{project-root}/.."
+    arm-types-dir: "{project-root}/../../common-types/resource-management"
```

Comment:
added to try to resolve the TypeSpec Validation check failure, but it did not help